src/main/java/com/patchworkmc/manifest/converter/ModManifestConverter.java

```diff
@@ -105,4 +110,16 @@ private static JsonArray convertAuthorsList(String authors) {
 
 		return array;
 	}
+
+	private static JsonObject getDependencies(ModManifest manifest, ModManifestEntry mod, boolean mandatory) {
+		JsonObject deps = new JsonObject();
+		Map<String, List<ModManifestDependency>> dependencyMap = manifest.getDependencyMapping();
+		dependencyMap.get(mod.getModId()).forEach(c -> {
+			if (mandatory) {
```

Comment:
Uh... Did you mean to compare this with the entry's mandatory status?

Reply:
Yup...